templates/unminified/cache.tpl.js

```diff
@@ -0,0 +1,125 @@
+(function() {
+  'use strict';
+
+  if (!('URLSearchParams' in window)) {
+    // Simple polyfill for URLSearchParams#has() and URLSearchParams#get().
+    class URLSearchParamsPolyfill {
```

Comment:
If we use es6 classes (or other es6 feature not present in webkit browsers) will require a transpilation in the process, forcing people to add that step if they don't have it.